src/qml/editorwidgets/RelationEditor/ordered_relation_editor.qml

```diff
@@ -0,0 +1,545 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQml.Models 2.15
+import QtGraphicalEffects 1.0
+import QtQuick.Layouts 1.12
+
+import org.qfield 1.0
+import org.qgis 1.0
+import Theme 1.0
+
+import "../.."
+import ".."
+
+EditorWidgetBase {
+    id: relationEditor
+
+    property int itemHeight: 40
+
+    // because no additional addEntry item on readOnly (isEnabled false)
+    height: listView.contentHeight
+              + (addEntry.visible ? addEntry.height : 0)
+    enabled: true
+
+    Rectangle {
+        anchors.fill: parent
+        color: "transparent"
+        border.color: 'lightgray'
+        border.width: 1
+    }
+
+    OrderedRelationModel {
+        //containing the current (parent) feature, the relation to the children
+        //and the relation from the children to the other parent (if it's nm and cardinality is set)
+        id: orderedRelationModel
+        currentRelationId: relationId
+        feature: currentFeature
+        orderingField: relationEditorWidgetConfig['ordering_field']
+        imagePath: relationEditorWidgetConfig['image_path']
+        description: relationEditorWidgetConfig['description']
+
+        property int featureFocus: -1
+
+        onFailedReorder: {
+          displayToast( "Failed to reorder features.", "error" )
+        }
+    }
+
+    DelegateModel {
+      id: visualModel
+
+      model: orderedRelationModel
+      delegate: dragDelegate
+    }
+
+    //the list
```

Comment:
I think we could get rid of these hyper-generic comment, or rename it to something more specific.

Reply:
```suggestion
    // The List!
```
:)))